package.json

```diff
@@ -51,6 +51,7 @@
   },
   "devDependencies": {
     "@salesforce-ux/design-system": "2.0.3",
+    "bluebird": "^3.4.7",
```

Comment:
Hum, we already have a dependency on "q", which is pretty similar to bluebird. Any chance we could skip having to download yet another dependency and instead use q ?